sql/src/Main.sk

```diff
@@ -54,6 +54,47 @@ fun queryParams(options: SKDB.Options): Map<String, P.Value> {
   }
 }
 
+// Read a single line from stdin, decode it to a JSON Object, expected
+// to be a map from table names to corresponding schemas (both strings)
+fun readSchemas(): Map<String, Array<P.ColumnDefinition>> {
```

Comment:
For consistency, might be worth moving this to the top of SqlEval, where we have other similar functions.